### PR TITLE
feat: create a bot to check on us

### PR DESCRIPTION
**Pull request by steveoh**
No one enjoys the act of nagging - so I am proposing we make a bot to nag for us and also keep us honest. 

We can mostly pull all the code from other projects we have already built to create this nag bot. Then we can schedule it to run every few days.

Comment if this is a good, bad, or otherwise idea.

If you would like to contribute to this project, let's do it.

### CODE REVIEW

**Comment by nathankota** [comment]
Nag bot...I love it. Prompting action on some of the cemetery issues has been a bit of a struggle, so I think it's worth it.

**Comment by steveoh** [comment]
I'm ready for a review

```
Name                           Stmts   Miss Branch BrPart  Cover
----------------------------------------------------------------
src/conductor/__init__.py          0      0      0      0   100%
src/conductor/checks.py          204      3     70      2    98%
src/conductor/conductor.py       115     12     40      1    92%
src/conductor/connections.py       3      0      0      0   100%
----------------------------------------------------------------
TOTAL                            322     15    110      3    96%
```

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "editor.formatOnSave": true,
+  "editor.rulers": [120],
+  "coverage-gutters.showGutterCoverage": false,
+  "coverage-gutters.showLineCoverage": true,
+  "coverage-gutters.showRulerCoverage": false,
+  "coverage-gutters.highlightdark": "rgb(61, 153, 112, .05)",
+  "coverage-gutters.noHighlightDark": "rgb(255, 65, 54, .05)",
+  "coverage-gutters.partialHighlightDark": "rgb(255, 133, 27, .05)",
+  "python.pythonPath": "/Users/sgourley/dev/porter/.env/bin/python3"
```
Comment (stdavis) [inline .vscode/settings.json:37]: I think that VSCode supports removing this project setting now. It's stored somewhere else.

File: src/conductor/conductor.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+a description of what this module does.
+this file is for testing linting...
+"""
```
Comment (stdavis) [inline src/conductor/conductor.py:6]: **chore**: docs

File: tests/test_checks.py [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+        'Data Source': True,
+        'Deprecated': False,
+    }
+    grade = GSheetChecker('fake.table', 'id', 'name',
+                          testing=True).grade(add=False, report_value=SheetResponse(True, grades))
+
+    assert grade == ' |\n| - deprecation issue link | :-1:'
+
+
+# def test_sheet_grading_sub_table():
```
Comment (stdavis) [inline tests/test_checks.py:498]: **chore**: remove

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+setup.py
+A module that installs projectname as a module
```
Comment (stdavis) [inline setup.py:5]: **chore**: docs

File: .vscode/settings.json (hunk 1 above)
Comment (steveoh) [reply .vscode/settings.json]: https://github.com/microsoft/vscode-python/wiki/AB-Experiments

**Comment by steveoh** [comment]
to all the reviewers tagged here... I'm planning to have a meeting to interactively code review this tool to help us with the porter process. 

If you would like to participate, please comment here and we can work out availability offline. We can record it as well if you think that would be of use.

**Comment by stdavis** [comment]
I'm interested.

**Comment by rkelson** [comment]
i'm interested

**Comment by nathankota** [comment]
I somehow missed this request for interest earlier. Interested, but will watch the recording.

**Pull request merged**